frontend/chrome-extension/background/src/main/kotlin/com/jakewharton/sdksearch/background/Background.kt

```diff
@@ -33,4 +35,14 @@ fun main(vararg args: String) {
 
     SearchUiBinder(presenter.events, Chrome, config.dacUrl).bindTo(presenter)
   }
+
+  Chrome.runtime.onInstalled.addListener { details ->
+    if (details["reason"] == "install") {
+      Chrome.management.get(THE_OTHER_SDK_SEARCH_ID) {
```

Comment:
Oh I didn't even know this API existed!

I was going to try using https://developer.chrome.com/extensions/runtime#method-connect, but I don't know if it works across extensions.

Reply:
I'll give it a try, it that works I could get rid of the `management` permission

Reply:
It seems to always return a `Port` instance even if you type in a garbage extension ID. I don't see a way to determine if the `Port` is actually connected or not.

Reply:
no luck